api/src/test/scala/org/ensime/api/EnsimeTestData.scala

```diff
@@ -53,15 +54,15 @@ trait EnsimeTestData {
   val file4 = canon("/foo/def")
   val file5 = canon("/foo/hij")
 
-  val refactorDiffEffect = new RefactorDiffEffect(9, RefactorType.AddImport, file2)
+  val refactorDiffEffect = new RefactorDiffEffect(9, RefactorType.AddImport, file2.file.toFile)
 
   val sourcePos1 = new LineSourcePosition(file1, 57)
   val sourcePos2 = new LineSourcePosition(file1, 59)
   val sourcePos3 = new EmptySourcePosition()
   val sourcePos4 = new OffsetSourcePosition(file1, 456)
 
-  val breakPoint1 = new Breakpoint(sourcePos1.file, sourcePos1.line)
-  val breakPoint2 = new Breakpoint(sourcePos2.file, sourcePos2.line)
+  val breakPoint1 = new Breakpoint(file1.file.toFile, sourcePos1.line)
```

Comment:
```file1.file.toFile``` really does not make the intention clear here - is there a better way to refactor this to capture the effective ```soucePos1.file```?